worker/backend/cni_network.go

```diff
@@ -52,6 +43,21 @@ const (
 	binariesDir = "/usr/local/concourse/bin"
 )
 
+var (
+	// nameServers is the default set of nameservers used.
+	//
+	nameServers = []string{"8.8.8.8"}
```

Comment:
Nit: for consistency with `defaultCNINetworkConfig`

```suggestion
	defaultNameServers = []string{"8.8.8.8"}
```

Reply:
7e2246a76f9c6c66f2a31fc7b3faf066b7bc6d45